pkg/web/templates/partials/secure_envelope/secure_envelope_list.html

```diff
@@ -100,6 +104,16 @@ <h3 class="font-semibold">Secure Envelope Meta Data</h3>
               </dd>
               {{ end }}
             </div>
+            {{ if .Error }}
+            <div class="py-1 grid grid-cols-2">
```

Comment:
I haven't added `Repair` yet as I wasn't quite sure how exactly we wanted to display it in the secure envelope list. It can be included in the story for implementing repair.